lib/sqs/sqs-base.js

```diff
@@ -24,6 +24,14 @@ const DEFAULT_BASE_AWS_EXT_CONFIG = {
       sqsOptions: {
         region: 'us-west-2'
       },
+      consumer: {
+        scheduler: {
+          scheduled: false,
+          start: '00:00:00',
+          duration: 2 * 60 * 60, // 2hrs
+          maxWaitSeconds: 6 * 60 * 60 // 6hrs
```

Comment:
Should this be maxVisibilityTimeout ?  I did not see reference for maxWaitSeconds in the code